nvflare/private/fed/server/job_cmds.py

```diff
@@ -162,7 +169,13 @@ def get_spec(self):
                     handler_func=self.pull_file,
                     authz_func=self.authorize_job_file,
                     client_cmd=ftd.PULL_BINARY_FQN,
-                    visible=False,
+                ),
```

Comment:
do we want download_job_file to be visible? is it still an internal command?

Reply:
It must stay invisible since it is used internally and cannot be issued by the user.

Reply:
got it, so we should not remove visible=False